web-ui/src/main/resources/catalog/js/admin/UserGroupController.js

```diff
@@ -339,13 +340,17 @@
        * @returns {boolean}
        */
       $scope.hideResetPassword = function() {
-        if ((!$scope.userSelected) ||
-            (!$scope.userSelected.security)) {
+        if ((!$scope.userSelected)
+            || (!$scope.userSelected.security)) {
+          return true;
+        } else if ($scope.userSelected.security.authType == 'LDAP') {
+          return true;
+        } else if ($scope.allowAdminReset && $scope.user.isAdministratorOrMore()) {
+          return false;
```

Comment:
Should not return `true` in this case?

Reply:
The function is hideResetPassword, so if allowAdminReset is true and user is admin, we don't want to hide it. Sounds correct ?

Reply:
Sounds fine, I should check the function name first.